pkg/test/framework/components/echo/config.go

```diff
@@ -163,6 +163,11 @@ func (c Config) IsNaked() bool {
 	return len(c.Subsets) > 0 && c.Subsets[0].Annotations != nil && !c.Subsets[0].Annotations.GetBool(SidecarInject)
 }
 
+func (c Config) IsTProxy() bool {
```

Comment:
what use cases do we have for tproxy today? I'm assuming some tests fail without this?

Reply:
it's pretty much to make tproxy count as a "special pod" so we won't filter it out (our default filter tries not to use multiple simple pods)